swarm/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.42.1 [unreleased]
```

Comment:
```suggestion
# 0.42.1
```

I can cut a new release of `libp2p-swarm` directly once merged. What do you think?

Reply:
I'd want to replace the banning API as well do make #2824 "complete" for now.